css/_recent_list.scss

```diff
@@ -0,0 +1,43 @@
+%recent-list-text {
+    width: 100%;
+    font-size: 14px;
+    line-height: 20px;
+    color: #FFF;
```

Comment:
It seems to me that `color: #FFF` is the only difference in this file compared to _navigate_section_list.scss. Is this correct?

If this is the case why do you need this file at all?

Reply:
- [ ] remove the recent_list css file